content/community/community-team/community-building-roles/contents.lr

```diff
@@ -13,54 +13,71 @@ Community building roles are for folks interested in helping grow CC's open sour
 * keep helping people in the CC open source community!
-  * respond to people on Slack
-  * respond to questions or issues on GitHub
-  * respond to people on the CC developer mailing list
+  * Respond to people on Slack.
+  * Respond to questions or issues on GitHub.
+  * Respond to people on CC's mailing lists.
 * participate in discussions in Slack or via email.
-* contribute in other ways if you’d like, like writing code or reviewing pull requests.
+* contribute in other ways if you’d like, like [writing code or reviewing pull requests](/community/community-team/project-roles/).
 
 ## Community Collaborator
-**Who should apply:** If you’ve helped a few people in the CC open source community and you’d like to do more to help grow the community, you should apply for this role!
+**Who should apply:**  
+If you’ve helped a few people in the CC open source community and you’d like to do more to help grow the community, you should apply for this role!
 
-**What does this role give you?**
+**What does this role give you?**  
 * Everything a Community Contributor gets.
-* Comment-only access to the Open Source Community project on Asana.
-  * You’ll see our future plans for the open source community and have the opportunity to participate in them.
+
+**How can you engage with the community?**  
+In addition to the channels afforded to a Community Contributor, you can use the following channels to engage with the community:
+* [Suggest access](/community/community-team/engagement-channels/#suggest-access)
+  on our upcoming tweets
 
 **Guidelines for Community Collaborators**  
 If you’ve been accepted as a Community Collaborator, you are encouraged to:
-* do everything a Community Contributor does
-* participate in the Asana project and do some tasks on there
-* suggest Tweet ideas for the [CC Open Source Twitter account](https://twitter.com/cc_opensource)
+* do everything a Community Contributor does.
+* suggest Tweet ideas for the [CC Open Source Twitter account](https://twitter.com/cc_opensource).
 
 ## Community Maintainer
-**Who should apply:**  If you’re a Community Collaborator who has done some meatier community building tasks and wants more responsibility, you should apply for this role. Please note that this role comes with a lot of responsibilities!
+**Who should apply:**  
+If you’re a Community Collaborator who has done some meatier community building tasks and wants more responsibility, you should apply for this role. Please note that this role comes with a lot of responsibilities!
 
-**What does this role give you?**
+**What does this role give you?**  
 * Everything a Community Collaborator gets.
-* Full access to the Open Source Community project on Asana.
-  * You’ll be able to add new tasks.
 * Manager permission on CC’s developer mailing list.
   * You’ll be able to approve new requests for people to join and moderate the list.
-* Access to tweet directly from CC Open Source account, if interested.
```

Comment:
I think this is still applicable, this is what you'll have.

Reply:
I removed this line because the Publish access a couple of lines later covers that.